config/routes.rb

```diff
@@ -7,4 +7,9 @@
   get '/commitments/:id', to: 'commitments#show', as: 'commitment'
   post '/commitments/list', to: 'commitments#list'
 
+  resources :failed_criteria, only: [:show]
+
+  resources :criteria, only: [:new, :create] do
+    resources :commitments, only: [:new]
```

Comment:
Conventional, but does `criteria/:id/commitments/new" make sense if a Criterium and Commitment instance are one-to-one?

Reply:
i think you're right here. it works fine for the new action, because we only care about the one record, but wouldn't work for any of the other actions (apart from create). so i've removed it and we'll need to send up a param with the criterium_id (we'll need it to link the commitment)